Formula/digdag.rb

```diff
@@ -0,0 +1,24 @@
+class Digdag < Formula
+  desc "Workload Automation System"
+  homepage "https://github.com/treasure-data/digdag"
+  url "https://dl.digdag.io/digdag-0.8.1.jar"
+  sha256 "92e79f5eb26b8992c9d510206c71f64fe4765c0c51a4e4cb62279aa1a4231ec7"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    # Add prefix a shebang before install for OS X 10.9.
+    # See https://github.com/Homebrew/homebrew-core/pull/2012#issuecomment-226407948 for the detail.
+    inreplace "digdag-#{version}.jar", /\A/, "#!/bin/sh\n"
```

Comment:
Now that I had more time thinking about this, I believe the best thing we can do is treat the `.jar` just like any other `.jar` file in our Java-using formulae and generate our own wrapper script. I cannot think of a formula where we do that off the top of my head and a quick search didn't reveal anything, thus not sure if this would be appropriate. Most Java-using software is distributed with a somewhat sane wrapper script where we tend to only modify the `JAVA_HOME` variable.